lib/httpSignatureStrategy.js

```diff
@@ -70,7 +70,25 @@ async function _getUser({keyDoc}) {
     }
     throw err;
   }
-  return identity;
+
+  const {identity, meta} = identityRecord;
+  const user = {identity};
+
+  // TODO: consider aliasing `user.actor` to `user.capabilities` or `user.ocaps`
+
+  // see if an account exists for the identity
+  if(typeof meta['bedrock-account'] === 'object' &&
+    typeof meta['bedrock-account'].account === 'string') {
+    const id = meta['bedrock-account.account'];
+    // identity is managed by `account`
+    user.account = await brAccount.get({actor: null, id});
```

Comment:
seems like an opportunity to parallelize.  Don't know what convention we should adopt, but this might work well here:
```
const account = brAccount.get({actor: null, id});
const actor = brAccount.getCapabilities({id: user.account.id});
user.account = await account;
user.actor = await actor;
```

Reply:
getCapabilities would have to just use `id` since user.account doesn't exist yet

Reply:
I'll see if it makes sense to go parallel. It would probably look something like this:

```js
[account, actor] = await Promise.all(
  brAccount.get(...),
  brAccount.getCapabilities(...));`
```